src/i18n/locales/en.json

```diff
@@ -85,6 +85,7 @@
   "emailAddress": "E-mail Address",
   "seatingCountValidationMessage": "Only positive numbers with a maximum of {{field}} digits are allowed.",
   "referenceNumberValidationMessage": "Only positive numerical values are allowed.",
+  "nameValidationMessage": "Only letters values are allowed.",
   "adminRegistrationApplications": {
```

Comment:
"Only letters values are allowed." cümle yapısı olarak yanlış bir de 

> [Minimum 2, maksimum 255 karakter uzunluğunda ve özel karakter içermeyen bir değer alabilir.](https://afetyonetimsistemi.atlassian.net/wiki/spaces/AYS/pages/32636949/Y+netim+Paneli+F1+-+Rol+Y+netimi+Analizi#Rol-Olu%C5%9Fturma)

Reply:
Hata mesajı içeriği güncellendi.

Reply:
"Role name should have minimum 2, maximum 255 characters. Special characters not allowed."

daha kısa ve daha açıklayıcı bence. bu arada linkteki yeni bilgiler ışığında validasyon değiştirilmesi gerekebilir.

Reply:
Hata mesajı daha kısa ve anlaşılır olacak şekilde düzenlendi.

Reply:
TR translationı da bu değişime göre güncellemek lazım, aklında bulunsun

Reply:
TR kısmı da tekrar güncellendi.